src/core/python/initialize.cpp

```diff
@@ -48,9 +49,13 @@ void initialize()
     py::register_exception_translator<stop_iteration_exception>(stop_iteration_translator);
 
 #ifdef HIPIPE_BUILD_PYTHON_OPENCV
-    // register OpenCV converters
+    // register OpenCV mat converters
     py::to_python_converter<cv::Mat, utility::matToNDArrayBoostConverter>();
     utility::matFromNDArrayBoostConverter();
+
+    // register OpenCV pt converters
```

Comment:
You resolved this conversation, but you have not replaced the `pt` by `point`, are you sure you have pushed all your commits?

Reply:
My bad